pennylane/numpy/tensor.py

```diff
@@ -121,6 +121,63 @@ def __getitem__(self, *args, **kwargs):
 
         return item
 
+    def __hash__(self):
+        if self.ndim == 0:

```

Comment:
Perhaps worth leaving a comment here too that `requires_grad` is preserved to resemble the behaviour of Torch?

Reply:
done!